main.js

```diff
@@ -19,15 +19,6 @@ if (config.jars) {
   jars = jars.concat(config.jars.split(":"));
 }
 
-if (config.pushConn && config.pushMidlet) {
-  MIDP.ConnectionRegistry.addConnection({
-    connection: config.pushConn,
-    midlet: config.pushMidlet,
-    filter: "*",
-    suiteId: "1"
-  });
-}
-
```

Comment:
I've removed them, I'm now using fgMidletClass. We might be able to remove the parameter altogether, will do in a followup.